developer/src/tike/project/Keyman.Developer.System.Project.kmnProjectFileAction.pas

```diff
@@ -80,8 +46,8 @@ function TkmnProjectFileAction.Clean: Boolean;
   begin
     FJS := TKeyboardUtils.GetKeymanWebCompiledFileName(FileName);
     CleanFile(FJS); // keyboard-x.y.js
-    CleanFile(ChangeFileExt(FJS, '') + '_load.js'); // keyboard-x.y_load.js
-    CleanFile(ChangeFileExt(FJS, '.json'), True); // keyboard-x.y_load.js
+//    CleanFile(ChangeFileExt(FJS, '') + '_load.js'); // keyboard-x.y_load.js
+//    CleanFile(ChangeFileExt(FJS, '.json'), True); // keyboard-x.y_load.js
```

Comment:
So we want to keep these files now?

Reply:
Those files are no longer generated -- they are both obsolete. I should probably remove the code altogether.

```suggestion
```